lib/components/src/tree-list/tree-list-item.scss

```diff
@@ -0,0 +1,69 @@
+.itemContent {
+    // .itemContent is a div so its width will be the width of its parent.
+    padding: 0px 32px 0px var(--indent);
+}
+.itemContent:hover {
+    cursor: pointer;
+    background-color: var(--mdc-theme-surface-neutral-highlight-hover);
+    border-radius: 0 16px 16px 0;
```

Comment:
Do we have a variable for the radius?

Reply:
Not defined in `_all-theme.scss` but should I define one in `tree-list-item.scss`?

Reply:
No, it would be a more global thing so don't worry about it